tests/components/nam/test_sensor.py

```diff
@@ -302,3 +302,30 @@ async def test_manual_update_entity(hass):
         )
 
     assert mock_get_data.call_count == 1
+
+
+async def test_unique_id_migration(hass):
+    """Test states of the unique_id migration."""
+    registry = er.async_get(hass)
+
+    registry.async_get_or_create(
+        SENSOR_DOMAIN,
+        DOMAIN,
+        "aa:bb:cc:dd:ee:ff-temperature",
+        suggested_object_id="nettigo_air_monitor_dht22_temperature",
+        disabled_by=None,
+    )
+
+    registry.async_get_or_create(
+        SENSOR_DOMAIN,
+        DOMAIN,
+        "aa:bb:cc:dd:ee:ff-humidity",
```

Comment:
This sensor is adjusted in the registry, but not used in this test.

Reply:
Fixed.